docs/superdesk-ninjs-schema.json

```diff
@@ -407,6 +407,10 @@
 			"items": {
 				"type" : "object",
 				"properties" : {
+					"id" : {
```

Comment:
what about using `code` here? that's more inline with other ninjs properties like place/person/subject/etc